tests/ansible_collections/conversion.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  tasks:
+    - name: Convert the system
+      shell: >
+        convert2rhel -y
+        --serverurl {{ rhsm_server_url | quote }}
```

Comment:
I am using `quote` because of advise here (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#notes)
```
To sanitize any variables passed to the shell module, you should use {{ var | quote }} instead of just {{ var }} to make sure they do not include evil things like semicolons.
```